.github/workflows/build.yml

```diff
@@ -113,6 +114,55 @@ jobs:
           message: |
             Build succeeded and deployed at http://${{ env.surge_url }} 
             (hash ${{ github.event.pull_request.head.sha }} deployed at ${{ steps.date.outputs.date }})
+
+  master_frontend_build_and_deploy:
+    needs: frontend_tests
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
Instead of using this and to keep things clean, you could also create a different file/workflo and use the "on:push:banches:master" as the only target.

Reply:
From our discussion: for this short term solution let's keep it here